frontend/src/components/App/PluginSettings/PluginSettings.stories.tsx

```diff
@@ -40,6 +41,18 @@ function createDemoData(arrSize: number, useHomepage?: boolean) {
   return pluginArr;
 }
 
+/**
+ * createDemoEnabledList function will create a list of plugin objects with a boolean value to enable/disable the plugin.
+ * The function will return an object of plugin names with a boolean value.
+ */
+function createDemoEnabledList(arr: PluginInfo[]): Record<string, boolean> {
+  const enabledList = arr.reduce((acc, p) => {
+    acc[p.name] = !!p.isEnabled;
+    return acc;
+  }, {} as Record<string, boolean>);
+  return enabledList;
```

Comment:
This seems overly complex for its purpose. Also it says it creates a list but it creates a record.